pythonz/installer/pythoninstaller.py

```diff
@@ -144,8 +144,6 @@ def __init__(self, version, options):
                 raise Exception
             if options.framework:
                 self.configure_options.append('--enable-framework=%s' % os.path.join(self.install_dir, 'Frameworks'))
-            if options.shared:
-                self.configure_options.append('--enable-shared')
```

Comment:
To be honest, I don't recall making that change. Can you cherry-pick that out, or would you like me to update the pull request.